flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniClusterJobDispatcher.java

```diff
@@ -358,12 +360,12 @@ private DetachedFinalizer(JobID jobID, int numJobManagersToWaitFor) {
 		}
 
 		@Override
-		public void jobFinished(JobExecutionResult result) {
+		public void jobFinished(org.apache.flink.runtime.jobmaster.JobExecutionResult result) {
 			decrementCheckAndCleanup();
 		}
 
 		@Override
-		public void jobFailed(Throwable cause) {
+		public void jobFailed(org.apache.flink.runtime.jobmaster.JobExecutionResult result) {
```

Comment:
Maybe rename to `JobResult` after all to avoid fqn.

Reply:
+1